From b61a0258d58a80704eabc1eb2df7298b4e733f70 Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Fri, 16 Nov 2018 18:07:03 -0800
Subject: [PATCH 8/9] sepol: Allow Settings to read
 ro.vendor.build.security_patch

qcom does this globally for every domain in their vendor sepolicy,
but devices not inheriting device/qcom/sepolicy need this.

Change-Id: I7aa147c7a3f1e56fda8dca6e219bd1452dd4af53
---
 common/private/system_app.te | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/common/private/system_app.te b/common/private/system_app.te
index 1b41d9a..e2c1f4b 100644
--- a/common/private/system_app.te
+++ b/common/private/system_app.te
@@ -1,3 +1,6 @@
 # For the updaters
 allow system_app cache_recovery_file:dir {add_name rw_file_perms};
 allow system_app cache_recovery_file:file {create rw_file_perms};
+
+# Allow Settings to read ro.vendor.build.security_patch
+get_prop(system_app, vendor_security_patch_level_prop)
-- 
2.17.1

